packages/components/src/components/Cell/Cell.tsx

```diff
@@ -48,20 +55,38 @@ export const Cell = React.forwardRef<View, React.PropsWithChildren<CellProps>>(
                     </View>}
                 </PressableHighlight>
             </View>
+        )
+        return (
+            (leftSwipeGroup || rightSwipeGroup) ?
+                <Swipeable
+                    overshootFriction={8}
+                    containerStyle={{ backgroundColor: styles.container.backgroundColor }}
+                    renderLeftActions={() => leftSwipeGroup && leftSwipeGroup.map((swipeItem, index) => (
+                        <CellSwipeItem key={`testL_${index}`} {...swipeItem} />
+                    ))}
+                    renderRightActions={() => rightSwipeGroup && rightSwipeGroup.map((swipeItem, index) => (
+                        <CellSwipeItem key={`test_${index}`} {...swipeItem} />
+                    ))}>
+                    {CellContent()}
+                </Swipeable>
+                :
+                CellContent()
```

Comment:
`CellContent()` should probably be changed to `<CellContent />`

Reply:
I had a chat with GPT yesterday about this after some confusions I had:

<img width="500" alt="Screenshot 2023-06-27 at 09 28 46" src="https://github.com/equinor/mad/assets/36899138/0f4d1864-f93a-4b50-bee5-d3c8714c52b1">

<img width="500" alt="Screenshot 2023-06-27 at 09 29 37" src="https://github.com/equinor/mad/assets/36899138/8d3ef562-601c-4464-bd01-56b273e28cd4">


What do you think? Should we enforce a standard on this? Personally, I like the {Component()} syntax for sub-components as it communicates that this is not an external component more clear, but I am open to doing things either way.

Reply:
Here is more info about `Component()` vs `<Component />`
https://dev.to/igor_bykov/react-calling-functional-components-as-functions-1d3l

Reply:
I think it should be left like it is for the moment. Since the component is stateless, there is no issues calling it like this. I have created #121 to track this.